Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -419,7 +419,11 @@ public final class ManifestLoader: ManifestLoaderProtocol {
             }
           #endif
             cmd += [resources.swiftCompiler.pathString]
+            // The interpreter doesn't work on Windows yet, so instead
+            // we compile and run it on Windows
+#if !os(Windows)
```

Comment:
Do you know if it'll be available soon? I think we should break this into cleaner functions for compiling vs interpreting and then always select the compiling one for windows.

Reply:
I don't think anyone is working on it at the moment. I can split them out though, that makes sense to me.

Reply:
Using `#if os(Windows)` for this means the code is likely to break as others do refactoring an Darwin/Linux. Can it instead be some global `Bool` whose default value differs on Windows, but which will allow us to test the behavior on Darwin/Linux as well?

Reply:
That's the plan (see my first comment).